Remotion/Data/DomainObjects/Persistence/Configuration/StorageProviderDefinition.cs

```diff
@@ -28,27 +27,20 @@ namespace Remotion.Data.DomainObjects.Persistence.Configuration
   /// Defines the configuration for a specific <see cref="StorageProvider"/>. Subclasses of <see cref="StorageProviderDefinition"/> can be 
   /// instantiated from a config file entry.
   /// </summary>
-  public abstract class StorageProviderDefinition: ExtendedProviderBase
+  public abstract class StorageProviderDefinition
   {
+    public string Name { get; }
+
```

Comment:
Order private before public. I know that it is kind of strange in property-field mixes if you want to retain the constructor order.

Reply:
actually, styleguide says public before private, that goes for state, too. Right now, there's not many cases where this is mixed. we should discuss this.
@a-ctor , @Frazerus